scripts/check-api-compatibility.sh

```diff
@@ -57,18 +74,23 @@ if ! ${SKIP_DOWNLOAD:-false}; then
 
     current_version=$(node -p 'require("./lerna.json").version')
     echo "Current version in lerna.json is $current_version"
+
+    rm -rf $tmpdir
+    mkdir -p $tmpdir
+
+    echo "Collecting peer dependencies" >&2
+    peer_deps=$(echo "$existing_names" | xargs -n1 -I {} bash -c 'peer_dependencies "$@"' _ $tmpdir {} | sort | uniq | tr '\n' ' ')
```

Comment:
Should this be:

```suggestion
    peer_deps=$(echo "$existing_names" | xargs -n1 -I {} bash -c 'peer_dependencies "$@"' _ $tmpdir {} | tr ' ' '\n' | sort | uniq | tr '\n' ' ')
```

peer_dependencies will output (potentially) multiple records per line, so you want to break into lines, and then run the `sort | uniq`, right? Or for that matter, just have `peer_dependencies` output newline-delimited?

Reply:
You're right. Good catch. Fixed.